README.md

```diff
@@ -76,6 +76,7 @@ App|Description
 App|Description
 ---|---
 [bus_scan](i2c/bus_scan) | Scan the I2C bus for devices and display results.
+[bmp280_i2c](i2c/bmp280_i2c) | Read and convert temperature and pressure data from a BMP280 sensor, attached to an I2C bus.
```

Comment:
`bm` is alphabetically before `bu` ?

Reply:
Indeed it is 😅

Reply:
Actually, while it makes sense for the files listed in `CMakeLists.txt` to be sorted alphabetically, it probably makes sense for the files listed in `README.md` to be sorted in order of increasing complexity? :thinking: 
In which case, `bus_scan` should indeed be listed before `bmp280_i2c` - sorry for changing my mind! :laughing: